lang.js

```diff
@@ -1,62 +1,32 @@
-'use strict'
-
-const fs = require('fs-extra')
+const LauncherDownload = require('minecraft-wrap').LauncherDownload
 const path = require('path')
-const getMinecraftFiles = require('./get_minecraft_files')
-
-if (process.argv.length !== 5) {
-  console.log('Usage : node lang.js <version1,version2,...> <output_dir> <temporary_dir>')
-  process.exit(1)
+const fs = require('fs')
+if (!(process.argv.length >= 3 && process.argv.length <= 4)) {
+  console.log('Usage : node lang.js <version>')
```

Comment:
can you make the output folder configurable ?

Reply:
yea

Reply:
and being able to specify several languages would also be quite convenient

Reply:
and versions

Reply:
okay well